src/main/java/com/tajweed/backend/model/Audio.java

```diff
@@ -0,0 +1,71 @@
+package com.tajweed.backend.model;
+
+import org.springframework.data.annotation.Id;
+
+public class Audio {
+
+    @Id
+    private String id;
+
+    private String duration;
+    private String idStudent;
+
+    private String idEntity;
+
+    private String chapter;
+
+    private String verse;
```

Comment:
I believe verse should be a range ? (what if a student does Al fatiha 1 -> 7 ?)

Reply:
that's right it has to be a range, verses were previously used for demo purposes